content/blog/mock-openings-practice-interviewing/index.md

```diff
@@ -28,4 +28,4 @@ For make it clear, we have some steps in this process:
 Your information, interviewing process, and your evaluation keep private with us.
 
 
-For starting, visit link [Mock Opening](/mock-openings/).
+For starting, visit link [Mock Opening](https://codingcoach.io/mock-openings).
```

Comment:
This link results in a 404 for me, but it seems to work with the trailing slash.  Try this instead?

```suggestion
For starting, visit link [Mock Opening](https://codingcoach.io/mock-openings/).
```